### PR TITLE
Link up tracing concepts in python manual instrumentation docs

### PR DESCRIPTION
**Pull request by cartermp**
Python joins JS, Java, and .NET in having links to language-agnostic concepts!

### CODE REVIEW

File: content/en/docs/instrumentation/python/manual.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ meter = metrics.get_meter(__name__)
 
 ### Creating spans
 
-To create a span, you'll typically want it to be started as the current span.
+To create a [span](/docs/concepts/signals/traces/#spans-in-opentelemetry, you'll
+typically want it to be started as the current span.
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:69]:
```suggestion
To create a [span](/docs/concepts/signals/traces/#spans-in-opentelemetry), you'll
typically want it to be started as the current span.
```

File: content/en/docs/instrumentation/python/manual.md [hunk 2]
```diff
@@ -81,7 +82,9 @@ span. This is usually done to track concurrent or asynchronous operations.
 ### Creating nested spans
 
 If you have a distinct sub-operation you'd like to track as a part of another
-one, you can create spans to represent the relationship:
+one, you can create
+[spans](/docs/concepts/signals/traces/#spans-in-opentelemetry to represent the
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:86]:
```suggestion
[spans](/docs/concepts/signals/traces/#spans-in-opentelemetry) to represent the
```

File: content/en/docs/instrumentation/python/manual.md [hunk 3]
```diff
@@ -102,8 +105,10 @@ nested span under `parent`.
 
 ### Creating spans with decorators
 
-It's common to have a single span track the execution of an entire function. In
-that scenario, there is a decorator you can use to reduce code:
+It's common to have a single
+[span](/docs/concepts/signals/traces/#spans-in-opentelemetry track the execution
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:109]:
```suggestion
[span](/docs/concepts/signals/traces/#spans-in-opentelemetry) track the execution
```

File: content/en/docs/instrumentation/python/manual.md [hunk 4]
```diff
@@ -123,8 +128,9 @@ use a decorator.
 
 ### Get the current span
 
-Sometimes it's helpful to access whatever the current span is at a point in time
-so that you can enrich it with more information.
+Sometimes it's helpful to access whatever the current
+[span](/docs/concepts/signals/traces/#spans-in-opentelemetry is at a point in
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:132]:
```suggestion
[span](/docs/concepts/signals/traces/#spans-in-opentelemetry) is at a point in
```

File: content/en/docs/instrumentation/python/manual.md [hunk 5]
```diff
@@ -135,8 +141,9 @@ current_span = trace.get_current_span()
 
 ### Add attributes to a span
 
-Attributes let you attach key/value pairs to a span so it carries more
-information about the current operation that it's tracking.
+[Attributes](/docs/concepts/signals/traces/#attributes) let you attach key/value
+pairs to a [span](/docs/concepts/signals/traces/#spans-in-opentelemetry so it
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:145]:
```suggestion
pairs to a [span](/docs/concepts/signals/traces/#spans-in-opentelemetry) so it
```

File: content/en/docs/instrumentation/python/manual.md [hunk 6]
```diff
@@ -150,8 +157,10 @@ current_span.set_attribute("operation.other-stuff", [1, 2, 3])
 
 ### Adding events
 
-An event is a human-readable message on a span that represents "something
-happening" during its lifetime. You can think of it as a primitive log.
+An [event](/docs/concepts/signals/traces/#span-events) is a human-readable
+message on a [span](/docs/concepts/signals/traces/#spans-in-opentelemetry that
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:161]:
```suggestion
message on a [span](/docs/concepts/signals/traces/#spans-in-opentelemetry) that
```

File: content/en/docs/instrumentation/python/manual.md [hunk 7]
```diff
@@ -167,8 +176,9 @@ current_span.add_event("Did it!")
 
 ### Adding links
 
-A span can be created with zero or more span links that causally link it to
-another span. A link needs a span context to be created.
+A [span](/docs/concepts/signals/traces/#spans-in-opentelemetry can be created
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:179]:
```suggestion
A [span](/docs/concepts/signals/traces/#spans-in-opentelemetry) can be created
```

File: content/en/docs/instrumentation/python/manual.md [hunk 8]
```diff
@@ -186,10 +196,11 @@ with tracer.start_as_current_span("new-span", links=[link_from_current]) as new_
 
 ### Set span status
 
-A status can be set on a span, typically used to specify that a span has not
-completed successfully - `StatusCode.ERROR`. In rare scenarios, you could
-override the Error status with `StatusCode.OK`, but don’t set `StatusCode.OK` on
-successfully-completed spans.
+A [status](/docs/concepts/signals/traces/#span-status) can be set on a
+[span](/docs/concepts/signals/traces/#spans-in-opentelemetry, typically used to
```
Comment (srikanthccv) [inline content/en/docs/instrumentation/python/manual.md:200]:
```suggestion
[span](/docs/concepts/signals/traces/#spans-in-opentelemetry), typically used to
```

**Comment by cartermp** [comment]
Thanks @srikanthccv - some errant copy/paste issue

**Review by srikanthccv — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]

**Pull request merged**